steps/cli/oc.go

```diff
@@ -171,5 +171,9 @@ var _ = gauge.Step("Configure the bundles resolver", func() {
 })
 
 var _ = gauge.Step("Create secrets for Tekton Results", func() {
-	oc.CreateSecretsForTektonResults()
+	if !oc.SecretExists("tekton-results-postgres", "openshift-pipelines") && !oc.SecretExists("tekton-results-tls", "openshift-pipelines"){
+		oc.CreateSecretsForTektonResults()
+	} else {
+		log.Printf("\"tekton-results-postgres\" and \"tekton-results-tls\" secrets already exist")
```

Comment:
it should say `or` instead of `and` :)

Reply:
The condition in the code was correct... what I meant is that `else` branch is effectively the negation of the condition. The negation of `and` is `or`.